src/generators/gencpp.ml

```diff
@@ -4657,6 +4667,22 @@ let gen_member_def ctx class_def is_static is_interface field =
             end;
          end;
          output "\n";
+	   | _ when has_class_field_flag field CfAbstract ->
+         let ctx_arg_list ctx arg_list prefix =
+            String.concat "," (List.map (fun (n,o,t) -> (ctx_arg ctx n None t prefix) ) arg_list)
+         in
+         let tl,tr = match follow field.cf_type with
+            | TFun(tl,tr) -> tl,tr
+            | _ -> die "" __LOC__
+         in
+         let return_type = (ctx_type_string ctx tr) in
+         let remap_name = native_field_name_remap is_static field in
+         output "virtual ";
+         output (if return_type="Void" then "void" else return_type );
+         output (" " ^ remap_name ^ "(" );
+         output (ctx_arg_list ctx tl "" );
+         output ") = 0;\n";
+         output ("		virtual ::Dynamic " ^ remap_name ^ "_dyn();\n" );
```

Comment:
I don't think this function should be virtual, since internally it will call a virtual function.
Then, it is not needed if the "doDynamic" logic ` (nonVirtual || not (is_override field ) ) && (reflective class_def field )` is not required.  This would be the case when one declared abstract method also exists in a parent class, or it is declared "@:unreflective"

Reply:
Right, good point. I've changed that in https://github.com/HaxeFoundation/haxe/pull/9716/commits/7010960fb65e6d277b310c5f11969d3c63935d44